.gitignore

```diff
@@ -5,3 +5,5 @@ _*/
 *.sublime*
 *~
 *.pyc
+.idea
+.idea/*
```

Comment:
I believe it's better to put editor-specific gitignores to `~/.gitignore`, however I see sublime there too...